hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson

```diff
@@ -0,0 +1,159 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "sysrst_ctrl"
+  // TODO: remove the common testplans if not applicable
+  import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
+  testpoints: [
+    {
+      name: smoke
+      desc: '''
+            Verify end to end data transfer in normal operation mode.
+            * Write a random data to the input keys.
```

Comment:
Add empty newline after the 'header' line for all testpoints.

Reply:
Also, some of the 'header' lines seem to be part of the bullet point list (`*`), which I believe is not correct. Please remove the `*` for the header lines.

Reply:
Added a new line after header line and removed a bullet point for each header line.